examples/posixio/posixio.c

```diff
@@ -12,7 +12,7 @@ int main(int argc, char **argv)
 
 
    /* Create a new file */
-   fd0 = creat("out.dat", 0655);
+   fd0 = create("out.dat", 0655);
```

Comment:
Is this change correct? The UNIX system call to create a file omits the e: https://linux.die.net/man/2/creat

Reply:
Thanks for catching this!